### PR TITLE
Deployment of the app to Azure

### PR DESCRIPTION
**Pull request by woocashh**

### CODE REVIEW

**Review by egroge — CHANGES_REQUESTED** [review]
We shouldn't have the build folder in git
Also, is it possible to have backend code one directory down? Adam had the idea to move it all out again in the pipeline if necessary

File: .github/workflows/deployment_daedalusproject.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        run: |
+          cd web
+          npm run build
+          
+      - name: Nuke node_modules
+        run: |
+          cd web
+          rm -rf node_modules
+
+      - name: Upload artifact for deployment jobs
```
Comment (TheAdamG) [inline .github/workflows/deployment_daedalusproject.yml:55]: Can we make it so this only happens on master? 

Is there anyway we could specify certain branches for deployment and have them deploy to different urls based on the branch name?

File: app.py [hunk 2]
```diff
@@ -30,3 +30,31 @@ def get_pcd(filename: str) -> Response:
...
         except FileNotFoundError:
             pass
+
+
+@bp.route("/", methods=["GET"])
+def default() -> Any: # TODO fix
+
+    print("APP")
+    return render_template("index.html")
+
```
Comment (TheAdamG) [inline app.py:40]: This file was created to only handle serving PCD, image and height map files to/from the front end, hence the flask "files" blueprint. 

Can we move the serving of the index.html to a different file and create a new blueprint for handling the routing for that?

**Review by TheAdamG — CHANGES_REQUESTED** [review]
Move back-end code back to the API folder, make sure deployment only occurs on at least master (would be nice if we could have different deployments to different URL's depending on the branch name), and refactor the app.py back to a files.py to only handle uploading/downloading files to/from the server, not to also serve the front-end

**Review by egroge — APPROVED** [review]

**Review by TheAdamG — APPROVED** [review]

**Pull request merged**